script.js

```diff
@@ -0,0 +1,690 @@
+      // make a single card object variable
+      const cardInfo = {
+        name: cardName,
+        suit: currentSuit,
+        rank: rankCounter,
+        symbol: suitSymbol,
+        color: colorType,
+        display: displayName,
+      };
+
+      // add the card to the deck
+      newDeck.push(cardInfo);
+    }
+  }
+
+  return newDeck;
+};
+// make deck
+let deck = shuffleCards(makeDeck());
+
+const loadDeck = () => {
+  deck = shuffleCards(makeDeck());
+};
+/* <==== Game element we need / what the user sees ===>
+1. Username input box (Done)
+2. Submit button (done)
+3. Bid amount input box (done)
+4. Display of 5 Cards + Card Container (done)
+5. Winnings shown on screen (done)
+6. Click to deal button (done)
+*/
+
+// Creating the elements on screen
+
+// output message div
+const outputMessageDiv = document.createElement('div');
+outputMessageDiv.setAttribute('class', 'display');
+
+// output Message helper function
+const outputMessage = (message) => {
+  outputMessageDiv.innerHTML = message;
+};
+
+// First screen display div to get user input for name
+const firstDisplayDiv = document.createElement('div');
+firstDisplayDiv.setAttribute('class', 'display');
+firstDisplayDiv.setAttribute('id', 'first-display');
+
+// Main display div to display all the elements
+const mainDisplayDiv = document.createElement('div');
+mainDisplayDiv.setAttribute('class', 'display');
+mainDisplayDiv.setAttribute('id', 'second-display');
+
+// bid amount display and label
+const bidAmountInput = document.createElement('input');
+bidAmountInput.setAttribute('type', 'number');
+bidAmountInput.classList.add('display', 'input-text');
+bidAmountInput.setAttribute('id', 'bid-amount-user');
+bidAmountInput.value = bid;
+const bidAmountLabel = document.createElement('label');
+bidAmountLabel.setAttribute('class', 'display');
+bidAmountLabel.innerText = 'Bid Amount:';
+
+// create br
+const brEL = document.createElement('br');
+
+// winnings display and label
+const displayWinnings = document.createElement('input');
+displayWinnings.setAttribute('type', 'number');
+displayWinnings.classList.add('display', 'input-text');
+displayWinnings.setAttribute('id', 'winnings');
+displayWinnings.value = winnings;
+// disabling the input ability for the user
+displayWinnings.disabled = true;
+const displayWinningsLabel = document.createElement('label');
+displayWinningsLabel.setAttribute('class', 'display');
+displayWinningsLabel.innerText = 'Winnings:';
+
+// background Div display
+const cardContainer = document.createElement('div');
+cardContainer.classList.add('cardContainer');
+
+// creation of deal button element
+const dealButton = document.createElement('button');
+dealButton.classList.add('btn', 'btn-success', 'button');
+dealButton.innerText = 'Deal Cards';
+
+// creation of back of card element
+// const cardBack2 = document.createElement('img');
+// cardBack2.setAttribute('src', 'https://tinyurl.com/y4bvjuw9');
+// cardBack2.classList.add('cardback', 'card');
```

Comment:
Please remove commented-out code before submitting, or at least provide a reason why you've left it in here commented-out. Otherwise it's like leaving rubbish around the house.